AsyncDisplayKit/Layout/ASBaselineLayoutSpec.h

```diff
@@ -29,7 +29,7 @@ typedef NS_ENUM(NSUInteger, ASBaselineLayoutBaselineAlignment) {
  If the spec is created with a vertical direction, a child's vertical spacing will be measured from its
  baseline instead of from the child's bounding box.
 */
-@interface ASBaselineLayoutSpec : ASLayoutSpec <ASBaselineLayoutable>
+@interface ASBaselineLayoutSpec : ASLayoutSpec
```

Comment:
Can this be a subclass of ASStackLayoutSpec, or can it be eliminated by folding in the extra functionality into ASStackLayoutSpec directly?